resources/js/svir_viewer_d3_tree.js

```diff
@@ -95,9 +96,9 @@
             }
             $(function() {
                 $("#spinner-"+id).width(100).spinner({
-                    min: 0,
+                    min: -100,
                     max: 100,
-                    step: 0.01,
+                    step: 0.000001,
```

Comment:
I agree. I would like to collect opinions about this change of precision, from Marco and Chris, and then possibly change from spinner to text input.